lib/lib.rs

```diff
@@ -72,7 +72,8 @@ use rpcs::{
         GetStateRootHashResult, GetValidatorChangesResult, ListRpcsResult, PutDeployResult,
         QueryBalanceResult, QueryGlobalStateResult, SpeculativeExecResult,
     },
-    v1_5_0::{
+    v1_6_0::get_account::AccountIdentifier,
```

Comment:
Nit: this can be merged into the existing `v1_6_0::get_account` imports just below here.